app/services/db-export/billing-charge-categories-table-export.service.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+/**
+ * Used by the 'root.controller' currently to determine if we are pulling
+ * the billing_charge_categories table correctly
```

Comment:
We don't tend to worry about stating where something is used, especially when that usage is temporary as in this case.

I'd suggest

```suggestion
 * Fetches all records from the water.billing_charge_categories table
```